Ports/.port_include.sh

```diff
@@ -101,6 +102,17 @@ install_main_launcher() {
     fi
 }
 
+install_icon() {
+    command -v convert >/dev/null 2>&1
+    convert_exists=$?
+    if [ -n "$icon_file" ] && [ -n "$launcher_command" ] && [ "$convert_exists" == 0 ]; then
+	run convert "${icon_file}[0]" app-16x16.png
+	run convert "${icon_file}[1]" app-32x32.png
```

Comment:
These indices are file-specific. For example, ScummVM has a 32x32 version at index 0.

Reply:
Good point. I added some logic to detect the correct layers to use for each icon size.